meta-mender-core/classes/grub-mender-grubenv.bbclass

```diff
@@ -0,0 +1,7 @@
+GRUB_MENDER_GRUBENV_REV = "6ffc280e64dd7b6bbc52f7747609f11fbdd1a057"
```

Comment:
You need to update this checksum, right? The [last build](https://gitlab.com/Northern.tech/Mender/grub-mender-grubenv/-/jobs/546363759) has `aa7e8f8c76c6aca6dca1820aaa42dc2cbf9762a1`.

Reply:
I already fixed that with a direct push! :-) This feature branch is not tested in Gitlab (yet), and there was nothing new to review besides the checksum, so I skipped a PR for this simple change.